apps/src/templates/teacherNavigation/TeacherNavigationBar.tsx

```diff
@@ -27,6 +27,12 @@ const TeacherNavigationBar: React.FunctionComponent = () => {
     {value: string; text: string}[]
   >([]);
   const [selectedSectionId, setSelectedSectionId] = useState<string>('');
+  const [selectedOptionKey, setSelectedOptionKey] =
+    useState<string>('assessments');
```

Comment:
Note: this is hard-coded for the time being.  I will still need to hook this up to however the user enters this page.